src/backend/aspen/api/utils/__init__.py

```diff
@@ -26,6 +26,6 @@
 from aspen.api.utils.tsv_streamer import (  # noqa: F401
     FieldSeparatedStreamer,
     GenBankSubmissionFormTSVStreamer,
-    GisaidSubmissionFormTSVStreamer,
+    GisaidSubmissionFormCSVStreamer,
```

Comment:
Not sure whether is better to change the name. Can change it back if that's more desirable